clusterman/migration/worker.py

```diff
@@ -74,7 +82,11 @@ def _monitor_pool_health(manager: PoolManager, timeout: float, drained: Collecti
         draining_happened = draining_happened or not any(
             node.agent.agent_id == connector.get_agent_metadata(node.instance.ip_address).agent_id for node in drained
         )
-        if draining_happened and manager.is_capacity_satisfied() and not connector.get_unschedulable_pods():
+        if (
+            draining_happened
+            and manager.is_capacity_satisfied()
+            and not (check_pods and connector.get_unschedulable_pods())
```

Comment:
Looks little bit risky to me. Our migration will stop if there is any pod can't be scheduled. What do you think consider "PodUnschedulableReason" as well in your conditions?

Reply:
Good call, I'll look in what info that can give.